prow/cluster/jobs/istio/tools/istio.tools.yaml

```diff
@@ -0,0 +1,74 @@
+presubmits:
+
+  istio/tools:
+
+    - name: tools-lint
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - make
+              - lint
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+    - name: tools-build
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - go
```

Comment:
as a followup may add make targets to the tools repo as they are not already present.